### PR TITLE
Update README and docs post-22.10

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: deploy/alibaba-cloud/README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ This repository contains information about how to deploy NVIDIA Triton Inference
 
 # Demo Instruction
 ## Prepare a model repo directory in OSS
-Download the tensorflow inception model via [fetch_model.sh](https://github.com/triton-inference-server/server/blob/main/docs/examples/fetch_models.sh). Then using [ossutil](https://help.aliyun.com/document_detail/50452.html?spm=a2c4g.11186623.6.833.26d66d51dPEytI) , which is a command line tool to use OSS, to upload the model to a certain OSS dir as you want.
+Download the tensorflow inception model via [fetch_model.sh](https://github.com/triton-inference-server/server/blob/r22.10/docs/examples/fetch_models.sh). Then using [ossutil](https://help.aliyun.com/document_detail/50452.html?spm=a2c4g.11186623.6.833.26d66d51dPEytI) , which is a command line tool to use OSS, to upload the model to a certain OSS dir as you want.
```
Comment (tanmayv25) [inline deploy/alibaba-cloud/README.md:51]: Do we have to change 'main' to 'r22.10'?

File: deploy/alibaba-cloud/README.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ See the following resources to learn more about how to use Alibaba Cloud's OSS o
 
 
 # Known Issues
-- [Binary Tensor Data Extension](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_binary_data.md) is not fully supported yet, users want to use service with binary extension supported, it is only available in cn-shanghai region of PAI-EAS.
+- [Binary Tensor Data Extension](https://github.com/triton-inference-server/server/blob/r22.10/docs/protocol/extension_binary_data.md) is not fully supported yet, users want to use service with binary extension supported, it is only available in cn-shanghai region of PAI-EAS.
```
Comment (tanmayv25) [inline deploy/alibaba-cloud/README.md:160]: Here as well.

File: deploy/gke-marketplace-app/benchmark/README.md [hunk 3]
```diff
@@ -36,8 +36,8 @@
 
 First, we collect a set of TensorFlow and TensorRT models to compare:
 
-- Get [Distill Bert fine-tuned with Squad Q&A task](https://huggingface.co/distilbert-base-cased-distilled-squad/tree/main) from Huggingface. `wget https://huggingface.co/distilbert-base-cased-distilled-squad/blob/main/saved_model.tar.gz`
-- Get [Bert base fine-tuned with Squad Q&A task](https://huggingface.co/deepset/bert-base-cased-squad2/tree/main) from Huggingface `wget https://huggingface.co/deepset/bert-base-cased-squad2/blob/main/saved_model.tar.gz`
+- Get [Distill Bert fine-tuned with Squad Q&A task](https://huggingface.co/distilbert-base-cased-distilled-squad/tree/main) from Huggingface. `wget https://huggingface.co/distilbert-base-cased-distilled-squad/blob/r22.10/saved_model.tar.gz`
```
Comment (tanmayv25) [inline deploy/gke-marketplace-app/benchmark/README.md:39]: Same here

File: deploy/mlflow-triton-plugin/README.md [hunk 4]
```diff
@@ -66,7 +66,7 @@ OUTPUT1 is the element-wise subtraction of INPUT0 and INPUT1.
 ### Start Triton Inference Server in EXPLICIT mode
 
 The MLflow Triton plugin must work with a running Triton server, see
-[documentation](https://github.com/triton-inference-server/server/blob/main/docs/getting_started/quickstart.md)
+[documentation](https://github.com/triton-inference-server/server/blob/r22.10/docs/getting_started/quickstart.md)
```
Comment (tanmayv25) [inline deploy/mlflow-triton-plugin/README.md:69]: Same here. Are we going to point to latest release instead of dev branch on dev branch?

Comment (mc-nv) [reply deploy/mlflow-triton-plugin/README.md:69]: @tanmayv25 - ack will fix it

File: deploy/alibaba-cloud/README.md (hunk 1 above)
Comment (mc-nv) [reply deploy/alibaba-cloud/README.md]: Hi @tanmayv25 , thank you for you review, I've waived that commit. 
Could you please review it again

File: qa/L0_infer/test.sh [hunk 5]
```diff
@@ -96,6 +96,7 @@ else
     TRITON_DIR=${TRITON_DIR:="/opt/tritonserver"}
     SERVER=${TRITON_DIR}/bin/tritonserver
     BACKEND_DIR=${TRITON_DIR}/backends
+    export SERVER_LD_PRELOAD=/usr/lib/$(uname -m)-linux-gnu/libgomp.so.1
```
Comment (Tabrizian) [inline qa/L0_infer/test.sh:99]: Why is this line required?

Comment (mc-nv) [reply qa/L0_infer/test.sh:99]: Hi @Tabrizian  , 
Thank you for asking over provided change. 
We are using this variable to preload specific library, as you remember we had done similar job on MA side: 
https://github.com/triton-inference-server/model_analyzer/pull/543/files

Comment (Tabrizian) [reply qa/L0_infer/test.sh:99]: So this is required only for PyTorch models? The L0_infer models are fairly simple, that is why I was wondering why it is required.

**Review by Tabrizian — APPROVED** [review]
